From f628be9b7a72ecca20bd49a73d910ee370de5573 Mon Sep 17 00:00:00 2001
From: Fabien Spindler <Fabien.Spindler@inria.fr>
Date: Tue, 17 Mar 2015 20:49:42 +0100
Subject: [PATCH 1/4] Add install rule for package.xml, disable testing

---
 CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8c8b906..7b5188b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2311,7 +2311,8 @@ include(${VISP_CMAKE_MODULE_PATH}/VISPGenerateConfig.cmake)
 #    visp-config --cflags ...
 #----------------------------------------------------------------------
 include(${VISP_CMAKE_MODULE_PATH}/VISPGeneratePkgConfigScript.cmake)
-
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/visp)
 #----------------------------------------------------------------------
 # Propagation in sub dirs to build demo, example, test, tutorial
 #----------------------------------------------------------------------
@@ -2321,7 +2322,7 @@ if(BUILD_TESTS)
   # We use CDash set through CTestConfig.cmake file
   # Dashboards are sent to http://cdash.irisa.fr/CDash/index.php?project=ViSP
   #----------------------------------------------------------------------
-  enable_testing()
+  #enable_testing()
   include(Dart)
   mark_as_advanced(DART_ROOT)
   mark_as_advanced(BUILD_TESTING)
-- 
1.9.1

