From fca279a2efaa14d298af752268648967abed21df Mon Sep 17 00:00:00 2001
From: Fabien Spindler <Fabien.Spindler@inria.fr>
Date: Fri, 10 Feb 2017 14:57:10 +0100
Subject: [PATCH 1/2] Add install rule for package.xml and disable testing

---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 694e188..5f153f2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -400,11 +400,14 @@ VP_CHECK_PACKAGE(Strtof)
 # Dashboards are sent to http://cdash.irisa.fr/CDash/index.php?project=ViSP
 #----------------------------------------------------------------------
 if(BUILD_TESTS)
-  enable_testing()
+#  enable_testing()
   mark_as_advanced(DART_ROOT)
   mark_as_advanced(BUILD_TESTING)
 endif()
 
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/visp)
+
 #----------------------------------------------------------------------
 # Try to find doxygen for documentation generation
 # Use "make visp_doc" target to generate the documentation
-- 
2.7.4

